src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -1089,33 +1089,20 @@ bool ReconstructionEngine_sequentialSfM::findNextBestViews(
     }
   }
 
-  // The beginning of the incremental SfM is a well known risky and
-  // unstable step which has a big impact on the final result.
-  // The Bundle Adjustment is an intensive computing step so we only use it
-  // every N cameras.
-  // We make an exception for the first 'nbFirstUnstableCameras' cameras
-  // and perform a BA for each camera because it makes the results
-  // more stable and it's quite cheap because we have few data.
-  static const std::size_t nbFirstUnstableCameras = 30;
-
-  if(_sfmData.getPoses().size() < nbFirstUnstableCameras &&
+  if(_sfmData.getPoses().size() < _params.nbFirstUnstableCameras &&
      !out_selectedViewIds.empty())
   {
     // add images one by one to reconstruct the first cameras
     ALICEVISION_LOG_DEBUG("findNextBestViews: beginning of the incremental SfM" << std::endl
       << "Only the first image of the resection group is used." << std::endl
       << "\t- image view id : " << out_selectedViewIds.front() << std::endl
-      << "\t- # unstable poses : " << _sfmData.getPoses().size() << " / " << nbFirstUnstableCameras << std::endl);
+      << "\t- # unstable poses : " << _sfmData.getPoses().size() << " / " << _params.nbFirstUnstableCameras << std::endl);
 
     out_selectedViewIds.resize(1);
   }
 
-  // Limit to a maximum number of cameras added to ensure that
-  // we don't add too much data in one step without bundle adjustment.
-  static const std::size_t maxImagesPerGroup = 30;
-
-  if(out_selectedViewIds.size() > maxImagesPerGroup)
-    out_selectedViewIds.resize(maxImagesPerGroup);
+  if(out_selectedViewIds.size() > _params.maxImagesPerGroup)
```

Comment:
Need to keep one comment sentence to explain the goal for quick readability of this function (even if it's explain on the variable).
Same on the previous if.

Reply:
Done.